aws/OWNERS

```diff
@@ -1,3 +1,4 @@
 approvers:
   - jeffwan
   - mameshin
+  - PatrickXYS
```

Comment:
quick question, are we also doing a rebase over here? It seems like there are a lot of tests not related to kfp-tekton.

Reply:
Never mind I just saw the commits above.